src/api-v1.js

```diff
@@ -319,6 +322,9 @@ api.declare({
     workerType: workerType,
   });
 
+  let queue = new taskcluster.Queue({credentials: this.credentials});
```

Comment:
In order to match the existing structure of the provisioner, `queue` should be a module in the main.js and passed in as context to the API.

Reply:
I passed it to WorkerType, at which point the API didn't need it anymore.

Reply:
Did you mean to delete this line?  We don't use this queue instance any longer from what I can tell, but maybe I'm wrong.